lib/label_printer/label_printer/label/base_tube.rb

```diff
@@ -32,7 +32,7 @@ def round_label_bottom_line(tube)
       end
 
       def barcode(tube)
-        tube.ean13_barcode
+        tube.human_barcode
```

Comment:
Just wondering whether it would be more correct to use machine_barcode here (haven't tested it). 

Thinking back to a conversation about a similar problem I had with James in Slack on 9th July (pasted below!):

Katy  3:56 PM
Can someone who knows more about barcodes than me help with something?
- The stock_resource table in MLWH has a column 'labware_machine_barcode', which can't be null
- It's currently being populated with the ean13_barcode method on plate
- Foreign barcodes don't have an ean13 barcode
Option 1: Fill it with the human barcode if the ean13 barcode doesn't exist
Option 2: Change the MLWH column to allow nulls (edited) 

I'm thinking option 2, because there's another column that contains the human barcode already, but I guess making an existing column allow nulls might mess up some downstream processing?

James  3:58 PM
Use machine_barcode instead

It’ll mostly be the same as human_barcode now, but automatically falls back to the ean13_barcode for older plates

We should probably rebroadcast some of the plates created recently, as the intent is to provide a way of knowing what the plate will scan asin order to join onto tables from, eg. labwhere (edited) 

Katy  4:03 PM
OK so I just tested a Lighthouse-created plate and a sample manifest-created plate, in UAT, and machine_barcode gives me the human barcode even for the second plate

Is that correct

Whereas ean13_barcode gives me a different number

James  4:04 PM
Yep, we changed the format barcodes were printed in to code39, which means that the actual barcode contains the same number as the text printed on it

Katy  4:04 PM
OK great thanks

James  4:05 PM
If you do something plate Plate.first.machine_barcode you’ll see the ean13 (Theoretically at least, I wouldn’t be surprised if the first plate has no barcode)

Reply:
I took a punt. It was either machine_barcode or human_barcode guess I backed the wrong horse. Happy to change.